client/js/components/user/DpOrganisationList/DpOrganisationFormFields.vue

```diff
@@ -214,6 +214,17 @@
               </div>
             </div>
           </template>
+          <template>
+            <dp-checkbox
+              id="procedureCreatePermission"
+              class="my-1.5"
+              data-cy="orgaFormField:procedureCreatePermission"
+              :label="{
+                text: Translator.trans('organisation.procedure.create.permission'),
+                bold: true
+              }"
+              v-model="localOrganisation.attributes.has_paid" /> <!--the name of the variable will change (can_create_procedures)-->
```

Comment:
I hope it will not be in snail_case but camelCase (`canCreateProcedures`)

Reply:
yes, we discussed it with Catalina. She will do it in a camelCase

Reply:
not necessary. Just wanted to be sure.